### DISCUSSION TITLE
How to make the top bar visible always and not disappear after some time?

### DISCUSSION THREAD

**Discussion by shohan11d** [end-4/dots-hyprland#712] [Q&A] [answered]
Hello, I just installed the dots and i really like it. My only issue is the top bar shows up late after i press the super key. It gets hidden after some time and in order to bring it back i have to press the super key again. There is again few seconds delay.

I wanted to know how do i make the top bar always show and not get hidden after some time. 
I have been trying to find it in the AGS config folder but no luck so far. 
Thanks in advance.

**Comment by end-4** [comment, marked as answer]
it's called a crash lol
#708

**Reply by shohan11d** [reply]
Haha, thought it was a ags feature. After some time it will get hidden for want of minimalism. LoL. It seems its hyprland's problem but not AGS. Thanks. Really like the dots btw. 

FIX: Logging into gnome, uninstalling hyprland-git and reinstalling hyprland fixed the issue for me
